lib/howitzer/mail_adapters/mailgun.rb

```diff
@@ -89,7 +89,7 @@ def self.events
 
       def self.event_by(recipient, subject)
         events.to_h['items'].find do |hash|
-          hash['message']['recipients'].first == recipient && hash['message']['headers']['subject'] == subject
+          hash['message']['recipients'].first == recipient && subject === hash['message']['headers']['subject']
```

Comment:
Please add unit test to cover Regest with `context "when subject is Rexexp"`
https://github.com/strongqa/howitzer/blob/master/spec/unit/lib/mail_adapters/mailgun_spec.rb#L33

Reply:
resolved